Open Inference/projects/OpenInference.json

```diff
@@ -0,0 +1,159 @@
+{
+  "name" : "OpenInference",
+  "options" : {
+    "v" : 6,
+    "filterBitArray" : "ffffffffffffffffffffffffffffffff",
+    "showGrid" : true,
+    "isModeloProject" : true,
+    "description" : "",
```

Comment:
add a description here for the project.   Also have the project-description open in a small pane on the right of the project so it is visible right away.  See the GenAI project for an example (or the Quickstart for a Tiled example).
In it, mention that users will need to update the URL for the source to point to their Triton server  (like in the Test Source Tutorial project description) and any other mods they might need to make. You could also add that to the README.md file but that's not 100% necessary.